app/routes/_layout+/bookings.$bookingId.tsx

```diff
@@ -328,8 +329,16 @@ export default function BookingEditPage() {
         <BookingForm
           id={booking.id}
           name={booking.name}
-          startDate={booking.fromForDateInput || undefined}
-          endDate={booking.toForDateInput || undefined}
+          startDate={
+            booking.from
+              ? dateForDateTimeInputValue(new Date(booking.from))
```

Comment:
should be done on client side so that zone conversion will be automatic

Reply:
Clear!